strawberryfields/ops.py

```diff
@@ -1576,6 +1577,41 @@ def decompose(self, reg):
         return cmds
 
 
+class GraphEmbed(Decomposition):
+    r"""Embed a graph into an interferometer set up.
+
+    This operation uses the Takagi decomposition to decompose
+    a adjacency matrix into a sequence of squeezers and beamsplitters and
+    rotation gates.
+
+    Args:
+        A (array): an :math:`N\times N` complex or real symmetric matrix.
+    """
+    ns = None
+    def __init__(self, A, max_mean_photon=1.0, make_traceless=True, tol=6):
```

Comment:
Nope, 6 is correct. This is because the tolerance is tested using the `np.round()` function, which accepts the number of decimal places to round to (6 in this case). Basically equivalent to 1e-6 precision.

Reply:
ok, then this works differently than basically every other tolerance in SF. To avoid confusion, can we either restructure to not use `np.round` or rename to something like `tol_decimal_places`?